### PR TITLE
docs: Add public API docs for the new Sessions API

### PR DESCRIPTION
**Pull request by Swatinem**
My question would be: How can I generate these locally so I can actually look at the results? :-D

### CODE REVIEW

**Comment by Swatinem** [comment]
Some constraints that I have no idea how to express:

* Some KV pairs in the results should match the request parameters
* `totals` and `series` should have a matching set of keys
* the array length of `groups.X.series` should match the one of `intervals`.

**Comment by manuzope** [comment]
Hey @Swatinem, here's how you can run it locally: https://develop.sentry.dev/docs/api/#local-development.

To confirm, is this endpoint ready to be made public (i.e. are we committed to not changing the endpoint in the future) or is it an early access feature?

**Comment by Swatinem** [comment]
Its rather early-access I would say. We will use it for our own UI, but also start to test-drive this with some external consumers as well.

**Comment by manuzope** [comment]
Got it. I think it would be good to include a `Caution` in the endpoint description saying that this is an experimental endpoint and to tell people that the endpoint can change without notice. 

AFAIK, there isn't a way to express those constraints in Open API Spec 3.

Would you mind posting a screenshot of how it looks on the docs page once you've gotten it running locally? It'll help us review the change.

**Comment by Swatinem** [comment]
> Would you mind posting a screenshot of how it looks on the docs page

image

I tried to insert a few manual breaks, but it doesn’t seem to work. Am I expected to write markdown syntax in that JSON?
I think this is good enough for now from a docs perspective. I will take a look at the tests.

File: api-docs/paths/organizations/sessions.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        "description": "A free-form query that is applied as a filter.\nAn example query could be `release:\"1.1.0\" or release:\"1.2.0\"`.",
+        "required": false,
+        "schema": {
+          "type": "string"
+        }
+      },
+      {
+        "name": "interval",
+        "in": "query",
+        "description": "This is the resolution of the time series, given in a `\"<number><unit>\"` format.\nFor example `1h` for a one hour resolution. Possible units are `m` for minutes, `h` for hours, `d` for days and `w` for weeks.\nThe default resolution is `1h` and the minimum resolution is currently restricted to `1h` as well.",
```
Comment (matejminar) [inline api-docs/paths/organizations/sessions.json:84]: Should we mention here that the interval longer than 1d might lead to unexpected results? I don't remember what was to problem exactly.

File: api-docs/paths/organizations/sessions.json [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        "required": false,
+        "schema": {"type": "string"}
+      },
+      {
+        "name": "statsPeriod",
+        "in": "query",
+        "description": "This defines the range of the time series, in the same format as the `interval`, relative to now.\nIt defaults to `90d`.",
```
[the remark this answers is not in the document]
Comment (Swatinem) [reply api-docs/paths/organizations/sessions.json:91]: I think so, yes:

https://github.com/getsentry/sentry/blob/1023fa9522e2e12c96501475a95d65077079263c/src/sentry/api/utils.py#L26-L29

**Review by matejminar — APPROVED** [review]

**Pull request merged**